packages/strapi-plugin-users-permissions/cypress/integration/login_specs.js

```diff
@@ -0,0 +1,49 @@
+const frontLoadingDelay = Cypress.config('frontLoadingDelay');
+const userData = {
```

Comment:
Can you change these identifiers?

Reply:
It's weird it has been changed here https://github.com/strapi/strapi/blob/add-cypress/packages/strapi-plugin-users-permissions/cypress/integration/login_specs.js